apps/docs/src/theme/ui/Sibling/Sibling.styles.js

```diff
@@ -31,7 +31,9 @@ export default createStyles((theme, _params, getRef) => {
 		},
 
 		body: {
-			ref: getRef('body')
+			ref: getRef('body'),
+      marginRight: `${_params.type === "next" ? theme.space.mdPX.value : "0px"} !important`,
+      marginLeft: `${_params.type === "prev" ? theme.space.mdPX.value : "0px"} !important`
```

Comment:
formatting is weird

Reply:
What should be done here?

Reply:
You can leave as is, I can fix that later, no problem

Reply:
Ok!